include/af/arith.h

```diff
@@ -21,14 +21,32 @@ namespace af
     /// \param[in] lhs first input
     /// \param[in] rhs second input
     /// \return minimum of \p lhs and \p rhs
-    ///
+    /// \deprecated use af::minOf instead
```

Comment:
we can use \ref for function for a direct redirect to there. leaving a note here